docs/libcurl/opts/CURLINFO_EARLYDATA_SENT_T.md

```diff
@@ -0,0 +1,73 @@
+---
+c: Copyright (C) Daniel Stenberg, <daniel@haxx.se>, et al.
+SPDX-License-Identifier: curl
+Title: CURLINFO_EARLYDATA_SENT_T
+Section: 3
+Source: libcurl
+See-also:
+  - curl_easy_getinfo (3)
+  - curl_easy_setopt (3)
+Protocol:
+  - All
+Added-in: 8.11.0
+---
+
+# NAME
+
+CURLINFO_EARLYDATA_SENT_T - get the number of bytes sent as TLS earlydata
```

Comment:
RFC 8446 speaks of it as **early data**, that is using two words. So I figure we should as well:

```suggestion
CURLINFO_EARLYDATA_SENT_T - get the number of bytes sent as TLS early data
```

Reply:
ack